jujugui/static/gui/src/app/components/env-size-display/env-size-display.js

```diff
@@ -16,8 +16,9 @@ class EnvSizeDisplay extends React.Component {
     const view = e.currentTarget.dataset.view;
     this.props.appState.changeState({
       gui: {
-        machines: view === 'machine' ? '' : null,
-        status: view === 'status' ? '' : null
+        application: view === 'application' ? true : null,
+        machine: view === 'machine' ? true : null,
```

Comment:
I see you changed '' to true. It appears that state is expecting this to be an empty string explicitly so best to change it back instead of updating state and then hunting down all the other instances (if any) and updating them as well.

Reply:
When I changed it back, the tab switching stopped working?